### PR TITLE
Helm integration of GraphScope

### LINKED ISSUE #161: Helm integration of GraphScope
Tracking some helm issues / PR's here relating to this repo.

- [ ] integrate GraphScope with the latest stable release of helm (v3)

https://github.com/helm/helm/releases/tag/v3.0.0
https://helm.sh/docs/faq/

### PR DESCRIPTION
**Pull request by lidongze0629**
## What do these changes do?

Helm integration of GraphScope.

WIP:
- [x] Only cleanup graph/app resources in sess.close() with addr mode.
- [x] Add Helm Test (Test coordinator service availability)
- [x] `sess.info`  failed in `addr` mode.
- [x] add dangling timeout_seconds ( -1 will disabled dangling check)
- [x] Rich comments and documentation of Helm.
- [x] Add NOTES.txt
- [x] Test LoadBalancer service type

## Related issue number

Fixes #161

### CODE REVIEW

**Comment by sighingnow** [comment]
Could we reconnect to the coordinator after close the session (after this pr landed)?

**Comment by lidongze0629** [comment]
> Could we reconnect to the coordinator after close the session (after this pr landed)?

Yes, a new session id will be generated for each session connection.

**Comment by sighingnow** [comment]
> > Could we reconnect to the coordinator after close the session (after this pr landed)?
> 
> Yes, a new session id will be generated for each session connection.

Please document such changes.

File: charts/graphscope/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+GraphScope charts
+=================
+
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/graphscope)](https://artifacthub.io/packages/helm/graphscope/graphscope)
+
+[GraphScope](https://graphscope.io) is a unified distributed graph computing platform that provides a one-stop environment for performing diverse graph operations on a cluster of computers through a user-friendly Python interface. GraphScope makes multi-staged processing of large-scale graph data on compute clusters simple by combining several important pieces of Alibaba technology: including GRAPE, MaxGraph, and Graph-Learn (GL) for analytics, interactive, and graph neural networks (GNN) computation, respectively, and the [vineyard store](https://v6d.io/) that offers efficient in-memory data transfers.
+
+## Get Repo Info
+```shell
```
Comment (sighingnow) [inline charts/graphscope/README.md:9]: Leave a blank line between title and content.

File: charts/graphscope/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+# Helm 2
+$ helm install --name [RELEASE_NAME] graphscope/graphscope
+```
+See configuration below.
+See [*helm install*](https://helm.sh/docs/helm/helm_install/) for command documentation.
+
+
+## Get GraphScope Service Endpoint 
+```shell
```
Comment (sighingnow) [inline charts/graphscope/README.md:28]: Ditto.

File: charts/graphscope/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+## Configuration
+See [*Customizing the Chart Before Installing*](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](https://github.com/alibaba/GraphScope/blob/main/charts/graphscope/values.yaml), or run these configuration commands:
+```shell
+# Helm 2
+$ helm inspect values graphscope/graphscope
+
+# Helm 3
+$ helm show values graphscope/graphscope
+```
```
Comment (sighingnow) [inline charts/graphscope/README.md:76]: Leave a blank line between blocks in markdown just in cases that some render may won't work for such format.

**Review by sighingnow — APPROVED** [review]
Add a section about helm in https://github.com/alibaba/GraphScope/blob/main/docs/deployment.rst, and https://github.com/alibaba/GraphScope/blob/main/docs/zh/deployment.rst.

Document the change about "reconnectable session" as well.

File: charts/graphscope/templates/NOTES.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
+The GraphScope has been installed.
```
Comment (sighingnow) [inline charts/graphscope/templates/NOTES.txt:1]: has been "deployed".

**Comment by lidongze0629** [comment]
> > > Could we reconnect to the coordinator after close the session (after this pr landed)?
> > 
> > 
> > Yes, a new session id will be generated for each session connection.
> 
> Please document such changes.

document in session's addr params.

**Pull request merged**